layouts/partials/edit_post.html

```diff
@@ -0,0 +1,3 @@
+{{- if (.Param "editPostUrl") }}
+&nbsp;·&nbsp;<a href="{{ .Param "editPostUrl" }}/content/{{ .File.Path }}">{{ .Param "editPostText" | i18n "edit_post" | default "Edit" }}</a>
```

Comment:
I think that `/content/{{ .File.Path}}` should not be appended to the value of `editPostUrl`. For example, if someone wanted to use a `mailto` link, it would not make sense to append a path to the email address.

Reply:
@kdkasad can you explain why Edit post URL will contain a `mailto` link ?

Reply:
> This is useful for providing the reader with an easy way to find the proper place to suggest changes.

If the site author wants the user to send an email to a certain address in order to request changes, they might want to use a mailto link there. For example:
```
mailto:webmaster@example.site?subject=suggested%20changes%20for%20example.site
```

Reply:
The Edit link is focused here for open sourced websites which people can contribute to..

Reply:
I am open to making the href more flexible and the `/content` suffix mandatory for the `editPostUrl`.
I am not that experienced enough with the framework to know if there would be cased where it would not be `content` but maybe a custom path. If there is such an option this could be accommodated with this change.

Also, I thought about if there might be options where you would not like to have `.File.Path` as the variable per post but something different and if this should be configurable.
For now I'd suggest leaving it as is: `editPostUrl` + `"/content/"` (or `"/"` depending on paragraph above) + `.File.Path`